### DISCUSSION TITLE
Issues with device filterin

### DISCUSSION THREAD

**Discussion by dieterlind** [Zefau/ioBroker.jarvis#2643] [General]
Hello!

Since upgrading to 3.1.8 I have issues in filtering devices:
![grafik](https://github.com/Zefau/ioBroker.jarvis/assets/93987222/077a3f7a-402e-4f13-8db2-8ccd036c3823)

Expectition is that just devices with "Thermo" are shown ... but no filterhing happens. 
No matter what I do.

Any suggestions on what I can do??

Thanks!

**Comment by Zefau** [comment]
Should be fixed with v3.2, which is hopefully coming soon to latest repo.

**Reply by dieterlind** [reply]
I updated to last 3.2-beta today ... but filtering is still not working ..

**Reply by Zefau** [reply]
Works for me with latest v3.2. Please send some screenshots of how the filter is applied and please send a screenshot of F12 error console.

**Reply by mcuiobroker** [reply]
@dieterlind Die Filterung bezieht sich nicht nur auf den Geräte-Namen (ID) sondern auch auf den Anzeige-DP
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/7259558c-a5b6-40ea-8b91-3effe36d77d8)

![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/4e27f612-952b-4bea-95fe-17ca93d93a8c)

**Reply by Zefau** [reply]
Kann man im Dropdown aber präzisieren:

![grafik](https://github.com/Zefau/ioBroker.jarvis/assets/14368811/829eaf2b-e1e6-4e91-bb00-86d021e2ef0a)

**Reply by dieterlind** [reply]
After checking again it seems to work now ... no reason why I didn't directly afer upgrading ... thanks ...
